src/OrchardCore/OrchardCore.Abstractions/BackgroundJobs/HttpBackgroundJob.cs

```diff
@@ -0,0 +1,68 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using OrchardCore.BackgroundTasks;
+using OrchardCore.Environment.Shell.Models;
+using OrchardCore.Environment.Shell.Scope;
+
+namespace OrchardCore.BackgroundJobs;
+
+public static class HttpBackgroundJob
+{
+    /// <summary>
+    /// Executes a background job in an isolated <see cref="ShellScope"/> after the current HTTP request is completed.
+    /// </summary>
+    public static Task AfterEndOfRequestAsync(string jobName, Func<ShellScope, Task> job)
```

Comment:
`ExecuteAfterEndOfRequestAsync()` or `RunAfterEndOfRequestAsync`

Reply:
Hmm, just to not be too long, the usage being

    HttpBackgroundJob.AfterEndOfRequestAsync()

In place of

    HttpBackgroundJob.ExecuteAfterEndOfRequestAsync()

But why not, no closed opinion but for now waiting for other feedbacks

Reply:
We could rename the class to `HttpBackgroundJobRunner`, but again using verbs for methods is a best practice

Reply:
Okay I will think about it and then change the name ;)

Reply:
Okay done